src/main/java/Wolox/training/controllers/UserController.java

```diff
@@ -127,4 +142,16 @@ public void delete(@PathVariable Long id) {
         return books;
     }
 
+
+    @GetMapping("/findby-birthdate-and-name")
```

Comment:
The API REST conventions suggest avoiding the use of verbs in the URLs. You can use the user's path `/api/users` and pass the query params like this `/api/users?startDate=1900-01-01&endDate=1900-01-10` 
```suggestion
    @GetMapping
```